### ISSUE TITLE
ブレイブリーデフォルトFtS [00040000000DB600]

### ISSUE THREAD

**Issue by ohyocbz** [azahar-emu/compatibility-list#23]
### Prerequisites

- [X] I have checked there is no currently open report for this game. (An updated report can be made for games with previously closed reports).
- [X] I have read through the contribution guide [here](https://github.com/Lime3DS/compatibility-list/blob/master/CONTRIBUTING.md) from start to finish.
- [X] I have played through this game from start to finish only using official releases of the Lime3DS emulator.
- [X] The OpenGL renderer was used for my entire playthrough.

### Rating

Perfect

### TL;DR
You need to change some settings and tweaks to run the game smoothly without issues.
I'm using an `Auto window size` resolution, `Default` screen layout; no mods nor custom textures.
59-60 FPS on the title screen and main menu (X button); 29-30 FPS in the overworld and during battle.
Both are running at 99-100% full speed.
The game is updated to the latest version which is Ver. 1.1.
![ブレイブリーデフォルトFtS_11 11 24_22 24 28 676](https://github.com/user-attachments/assets/8f6e31bc-fa15-45b1-9bb3-ffabbb73e586)
![ブレイブリーデフォルトFtS_11 11 24_22 24 51 5](https://github.com/user-attachments/assets/f268e385-fe22-4abd-97f7-bbbf44ee20e0)

### Before launching
Go to _Emulation_ > _Configure..._ > _System_ > _System Settings_, change the _Language_ to `Japanese`.
_Country_ should be set to `Japan` to make the game work.
There's no need to change the Game ROM's emulation to `JPN` (from _Right-clicking the game name_ > _Properties_ > _General_ > _Emulation_). `Auto-select` will detect your current system settings.

If you saw the game intro **_NOT in Japanese_** around this part, it means you _**didn't configure the settings correctly**_ and Lime3DS will crash automatically:
![ブレイブリーデフォルトFtS_11 11 24_17 53 58 211](https://github.com/user-attachments/assets/cb408729-3752-42a9-bf4b-b7b95fd03b7d)

Following the instructions correctly, you'll get past the game intro and title screen:
![ブレイブリーデフォルトFtS_11 11 24_18 21 15 017](https://github.com/user-attachments/assets/57c0c0d8-7f07-428c-b44a-ffe5fc3029e7)
![ブレイブリーデフォルトFtS_11 11 24_18 13 56 083](https://github.com/user-attachments/assets/dcc9edbc-c83c-4478-a84d-6b1c07f7e40c)
![ブレイブリーデフォルトFtS_11 11 24_18 27 31 69](https://github.com/user-attachments/assets/5001790e-3fc2-4a08-8038-8bf617125845)

If your save file is not JPN region, it will override the _Username_ and _Country_ from Lime3DS _System Settings_ instead:
![ブレイブリーデフォルトFtS_11 11 24_18 36 56 66](https://github.com/user-attachments/assets/5f39c9bb-8127-41d9-b4fb-3a88cd1cb048)
> Example: EUR region save file from Bravely Default will 100% work in the FtS version since it's the same ROM but minor changes in the code such as the Alternate Costumes censorships

To fix the missing Mii icons, you need _errfix.3dsx_ (just Google it and you'll see that one Reddit post to download the file).
Put it in the same folder as your ROM and launch it.
It boots in a few seconds and return you to the main window.
Run the game and your Miis' faces are back to normal.

Before:
![ブレイブリーデフォルトFtS_11 11 24_18 35 21 862](https://github.com/user-attachments/assets/30938b50-099d-4f29-a145-15b542df77c1)

After:
![ブレイブリーデフォルトFtS_11 11 24_19 50 12 84](https://github.com/user-attachments/assets/92a101b9-2bca-4999-be6f-cac8ea270c98)
![ブレイブリーデフォルトFtS_11 11 24_19 50 25 434](https://github.com/user-attachments/assets/73ef82a6-81a8-483d-afa9-b98a0c6d5d88)
> My Mii icon is missing since I haven't dumped my _aes-keys.txt_ yet from my own 3DS to download MiiMaker (JPN) and re-create my own Mii.

![ブレイブリーデフォルトFtS_11 11 24_21 51 05 872](https://github.com/user-attachments/assets/c5379514-25b2-4a8f-ad43-b30a8cdaf07c)
> This is after re-creating my own Mii from MiiMaker (JPN).

### Lime3DS Version

2119

### System Configuration

CPU: Intel Core i7-1065G7
GPU/Driver: NVIDIA GeForce MX110 (Driver ver. 566.03)
RAM: 20GB DDR4 2666 MT/s (4GB onboard + 16GB)
OS: Windows 11 Home Single Language (24H2)

**Comment by Mateo-Git** [comment]
Can confirm **Bravely Default For The Sequel ブレイブリーデフォルトFtS [00040000000DB600]**, as well as its overseas release **Bravely Default [00040000000FC500]** (which already has a Perfect rating and is mostly identical to the Japanese version besides the different title and altered/removed costumes which can be restored via either romfs mods or save editing/cheats), work perfectly without any issues.

For the record, Its sequel **Bravely Second ブレイブリーセカンド [00040000000DC800]** (which uses the same engine as its predecessor) and its overseas release **Bravely Second [000400000017BA00]** (which already has a Great rating and also is mostly identical to the Japanese version besides the same differences and a change which only affects the result obtained on an in-game event), work perfectly as well.

The games don't really need any special configuration to run:

They will use the language can be specified on System settings to run, but it can be changed either via System settings or in-game via its config options at any time and will boot fine with auto-select/any region. (Crash other reporter mentioned may have happened due to missing system files, but they work perfectly on set-up Azahar)
No need for any external .3dsx files either if you set up your files from a real 3DS, the Mii icons will render perfectly.
They also run great with LLE either on/off.
This is out of the scope of the compatibility list, but they even run perfectly with/without romfs and 60FPS mods.

In short, these games run flawlessly on Azahar 2120+. They belong in the Perfect rating.
